.github/workflows/tiobe.yml

```diff
@@ -0,0 +1,35 @@
+name: Tiobe
+
+on:
+  schedule:
+    - cron: '0 0 * * TUE'
+
+jobs:
+  coverity:
+    if: github.repository == 'canonical/netplan'
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          echo "APT::Get::Always-Include-Phased-Updates \"true\";" | sudo tee /etc/apt/apt.conf.d/90phased-updates
+          sudo sed -i 's/Types: deb/Types: deb deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
+          sudo apt update
+          sudo apt build-dep -y netplan.io
+          sudo apt install -y gcovr python-is-python3 pylint flake8 python3-flake8
+      - name: Build Netplan
+        run: |
+          # The build directory must be called _build. TIOBE will look for coverage data in it.
+          meson setup _build --prefix=/usr -Db_coverage=true
+          meson compile -C _build -v
+          meson test -C _build -v
+      - name: Download and Install TICS
+        run: |
+          curl --silent --show-error "https://canonical.tiobe.com/tiobeweb/TICS/api/public/v1/fapi/installtics/Script?cfg=default&platform=linux&url=https://canonical.tiobe.com/tiobeweb/TICS/" > install_tics.sh
+          bash install_tics.sh
+      - name: Run TICS
+        run: |
+          export TICSAUTHTOKEN=${{ secrets.TICSAUTHTOKEN }}
+          source ~/.profile
+          TICSQServer -project netplan -branchname master -calc ALL -tmpdir /tmp/tics -branchdir .   # wokeignore:rule=master
```

Comment:
Another question: Is `-branchname master` correct? Our upstream branch is called `main`, but I do not know how TiCS works in detail, so this might still be correct.

Reply:
The name "master" seems to be used by the remote service receiving the data actually.